modules/admin-ui-frontend/app/scripts/shared/services/restServiceMonitor.js

```diff
@@ -45,6 +48,42 @@ angular.module('adminNg.services')
 
     Monitoring.getActiveMQStats();
     Monitoring.getBasicServiceStats();
+    Monitoring.getVersionStats();
+  };
+
+  Monitoring.getVersionStats = function() {
+    $http.get(MY_VERSION_PATH).then(function(response_my_version) {
+      $http.get(LATEST_VERSION_PATH).then(function(response_latest_version) {
+        Monitoring.populateService(LATEST_VERSION_NAME);
+
+        if (response_latest_version.status === 200 && response_my_version.status === 200
+        && response_my_version.data.consistent) {
+          var my_version = response_my_version.data.version;
+          var latest_version = response_latest_version.data;
+          services.service[LATEST_VERSION_NAME].docs_url =
+            'https://docs.opencast.org/r/' + latest_version[0] + '.x/admin/';
+
+          if (parseFloat(my_version) >= parseFloat(latest_version) || my_version.endsWith('SNAPSHOT')) {
```

Comment:
```suggestion
          if (parseFloat(my_version) >= parseFloat(latest_version)
             || (my_version[0] == latest_version[0] && my_version.endsWith('SNAPSHOT'))) {
```

This could also be `4.0.0-SNAPSHOT` which could be identified as very old.